crates/phactory/api/src/storage_sync.rs

```diff
@@ -21,10 +21,10 @@ pub enum Error {
     /// No Justification found in the last header
     MissingJustification,
     /// Header validation failed
-    #[display(fmt = "HeaderValidateFailed({})", .0)]
+    #[display(fmt = "HeaderValidateFailed({})", self)]
```

Comment:
It should be `_0` according to the [doc](https://jeltef.github.io/derive_more/derive_more/display.html#the-format-of-the-format).

`self` means the entire `err: Error`. Putting the `self` here would cause infinite recursion.

Reply:
Another commit.